math/src/field/fields/fft_friendly/babybear.rs

```diff
@@ -0,0 +1,101 @@
+use crate::{
+    field::{
+        element::FieldElement,
+        fields::montgomery_backed_prime_fields::{IsModulus, MontgomeryBackendPrimeField},
+    },
+    unsigned_integer::element::U64,
+};
+
+pub type U64MontgomeryBackendPrimeField<T> = MontgomeryBackendPrimeField<T, 1>;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct MontgomeryConfigBabybear31PrimeField;
+impl IsModulus<U64> for MontgomeryConfigBabybear31PrimeField {
+    //Babybear Prime p = 2^31 - 2^27 + 1 = 0x78000001
+    const MODULUS: U64 = U64::from_u64(2013265921);
+}
+
+pub type Babybear31PrimeField =
+    U64MontgomeryBackendPrimeField<MontgomeryConfigBabybear31PrimeField>;
+
+impl FieldElement<Babybear31PrimeField> {
+    pub fn to_bytes_le(&self) -> [u8; 8] {
+        let limbs = self.representative().limbs;
+        limbs[0].to_le_bytes()
+    }
+
+    pub fn to_bytes_be(&self) -> [u8; 8] {
+        let limbs = self.representative().limbs;
+        limbs[0].to_be_bytes()
+    }
+}
+
+impl PartialOrd for FieldElement<Babybear31PrimeField> {
+    fn partial_cmp(&self, other: &Self) -> Option<core::cmp::Ordering> {
+        self.representative().partial_cmp(&other.representative())
+    }
+}
+
+impl Ord for FieldElement<Babybear31PrimeField> {
+    fn cmp(&self, other: &Self) -> core::cmp::Ordering {
+        self.representative().cmp(&other.representative())
+    }
+}
```

Comment:
This isn't needed, but it's ok, we can document this later and remove if needed. Doesn't hurt either